app/state_machines/flood_risk_engine/step_machine.rb

```diff
@@ -21,15 +21,17 @@ def current_step
       state.to_s
     end
 
+    # TODO: rename this method to something better. Style guide recommends
+    # not using set_ at start of method
+    # rubocop:disable Style/AccessorMethodName
     def set_step_as(step)
       restore!(step.to_sym)
     end
+    # rubocop:enable Style/AccessorMethodName
```

Comment:
I think there is probably a better name for this method, but can't think of one at the moment.